task/sql-tasks.js

```diff
@@ -178,7 +266,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT

```

Comment:
it's better to use 2 select statements here, it's more descriptive. It's also possible to use select without from clause
e.g. 
```
select 
 (select count() from ....) as Total
 (select count() from ....) as TotalDiscontined
```